CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [ryu](https://github.com/dtolnay/ryu) is used to serialize `f32` and `f64`
 - Added missing implementation for `serialize_bytes` method
```

Comment:
We should put this line into an `###Added` section before we do a new release.

Reply:
I added the new section and re-organized the existing changelog to keep everything in line. Let me know if you'd prefer this to be in a different PR!

Reply:
Looks good thanks!